kernel/network.c

```diff
@@ -156,17 +156,15 @@ BYTE remote_getfree_11a3(void FAR *cds, void *dst)
     return SUCCESS;
 }
 
-BYTE remote_getvolumeinfo(UBYTE drive, UBYTE bufsiz, void *dst_r, char FAR *dst_s)
+BYTE remote_getvolumeinfo(UBYTE drive, struct xgetvolumeinfo *s)
 {
-    UWORD *udst = (UWORD *)dst_r;
     iregs regs = {};
-    struct xgetvolumeinfo s;
     struct xgetvolumeinfo FAR *p;
 
-    p = MK_FAR(s);
+    p = MK_FAR(*s);
 
     regs.a.x = 0x11a0;
-    regs.c.x = sizeof(s);
+    regs.c.x = sizeof(*s);
```

Comment:
So if that doesn't compile, you
can pass `s_ptr` and do
`*s_ptr = s;` at the end.

Reply:
It's compiling okay now, thanks.

Reply:
Good that I omitted porting this
horrible templated compiler to
dj64dev. :) Fixing the oddities like
this one, could be a nightmare.